packages/rosaenlg/src/LanguageFrench.ts

```diff
@@ -140,24 +142,28 @@ export class LanguageFrench extends LanguageImpl {
     return frenchParse(val, { dictHelper: this.dictHelper });
   }
 
-  thirdPossessionTriggerRef(owner: any, owned: any, params: any): void {
+  thirdPossessionTriggerRef(owner: any, owned: any, params: ValueParams): void {

```

Comment:
One possible parameter is, in English, `possForm`: can be `S` or `OF`. But it is not defined is `ValueParams`. Can you please check?
- either add `possForm` to `ValueParams`, if it makes sense
- or create a new type

Reply:
let's add it ot ValueParams